### PR TITLE
Best practice review

### PR DESCRIPTION
**Pull request by Mahdi-Niazi**
## Best Practice Review
In this milestone I:

- Add javascript files into one big file for review
- Add HTML file
- Add CSS files

### CODE REVIEW

File: javascript.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  }
+
+  return true;
+};
+
+
+const deleteTask = (i) => {
+  let tasks = JSON.parse(localStorage.getItem('datas'));
+  tasks = tasks.filter((task) => task.description !== tasks[i].description);
+  for (let i = 0; i < tasks.length; i += 1) {
```
Comment (Zel-hub7) [inline javascript.js:25]: So far so good, But I have something to add on this. Variable names should be meaningful and descriptive. places, I kindly request you to improve variable names for better readability. For example, i in the deleteTask and showTask functions could be renamed to taskIndex or something similar to make it clearer.

File: LICENSE
Comment (Zel-hub7) [inline LICENSE:1]: I believe you have taken your time doing this project and It is really awesome. But there are some small changes you gotta make to make it even more great. The code does not include comprehensive error handling. It's important to handle potential errors that may occur during parsing JSON, accessing elements by ID, or any other operations that could potentially fail. Adding appropriate error handling can improve the robustness of the code. So I kindly request you to add this feature.

**Review by Zel-hub7 — CHANGES_REQUESTED** [review]
Almost there.

File: LICENSE
Comment (Mahdi-Niazi) [reply LICENSE:1]: Dear @Zel-hub7, 

Kindly refer me to place where I need to bring this changes. 


Thank you

**Review by Zel-hub7 — APPROVED** [review]
Status Approved.

**Pull request merged**